kyo-core/shared/src/main/scala/kyo/logs.scala

```diff
@@ -58,53 +55,57 @@ object Logs:
     end Unsafe
 
     object Unsafe:
-        def apply(logger: org.slf4j.Logger): Unsafe = new Unsafe:
-            inline def traceEnabled: Boolean = logger.isTraceEnabled
-            inline def debugEnabled: Boolean = logger.isDebugEnabled
-            inline def infoEnabled: Boolean  = logger.isInfoEnabled
-            inline def warnEnabled: Boolean  = logger.isWarnEnabled
-            inline def errorEnabled: Boolean = logger.isErrorEnabled
+        class ConsoleLogger(name: String) extends Logs.Unsafe:
+            inline def traceEnabled: Boolean = true
```

Comment:
I figured if this is a class, users can extend and override these values.

Reply:
Nice!!